### PR TITLE
Multiple modules in support of POE-1.367

### PR DESCRIPTION
**Pull request by limeytexan**
POE: init at 1.367
POETestLoops: init at 1.360
TestPodNo404s: init at 0.02

Note that I encountered bugs as part of this effort as documented in the following CPAN bug tickets:

* https://rt.cpan.org/Ticket/Display.html?id=124495: POE-1.367 failed t/10_units/01_pod/03_pod_no404s.t
* https://rt.cpan.org/Ticket/Display.html?id=124496: POE-1.367 failed t/10_units/01_pod/04_pod_linkcheck.t

... and these bugs are addressed by the perl-POE-1.367-pod_no404s.patch and perl-POE-1.367-pod_linkcheck.patch files included with this patch, both of which have been submitted as PRs to https://github.com/rcaputo/poe.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/perl-packages.nix [hunk 1]
```diff
@@ -11297,6 +11297,47 @@ let self = _self // overrides; _self = with self; {
     };
   };
 
+  POE = buildPerlPackage rec {
+    name = "POE-1.367";
+    patches = [
+      ../development/perl-modules/perl-POE-1.367-pod_linkcheck.patch
+      ../development/perl-modules/perl-POE-1.367-pod_no404s.patch
```
Comment (Mic92) [inline pkgs/top-level/perl-packages.nix]: Do you have open pull requests for these?

**Comment by limeytexan** [comment]
Sorry for the newbie confusion - I will close this PR and resubmit as three individual PRs for each of the three modules.

**Comment by Mic92** [comment]
Multiple simple packages in one PR are ok. Your pull request just went under, which happen sometimes.

**Comment by limeytexan** [comment]
Thanks for letting me know. I'm happy resubmitting multiple PRs for related
modules in future as it is better suited to the automated handling and also
easier to monitor progress from my end.

Yes, I submitted the following PRs to the CPAN author so that hopefully
those patches will be applied upstream at some point in future:

   - https://github.com/rcaputo/poe/pull/25
   - https://github.com/rcaputo/poe/pull/26

FYI, I will be submitting a new PR for the POE module once the current PR
for POE::Test::Loops (https://github.com/NixOS/nixpkgs/pull/36115) is
successfully merged.

Thanks again!

**Pull request closed without merging**